src/main/webapp/feed.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+  <title>Message Feed</title>
+  <link rel="stylesheet" href="/css/main.css">
+  <link rel="stylesheet" href="/css/user-page.css">
+
+  <script>
+    // Fetch messages and add them to the page.
+    function fetchMessages() {
+      const url = '/feed';
+      fetch(url).then((response) => { return response.json() })
+      .then((messages) => {
```

Comment:
Lines which wrap over multiple lines (including chained methods) should be indented:

```
fetch(url)
  .then(...)
  .then(...)
```

Reply:
(You can also install a code formatter like Prettier to fix this automatically.)